src/mono/wasm/wasm.proj

```diff
@@ -196,7 +196,6 @@
       <EmccExportedRuntimeMethod Include="FS_createDataFile" />
       <EmccExportedRuntimeMethod Include="removeRunDependency" />
       <EmccExportedRuntimeMethod Include="addRunDependency" />
-      <EmccExportedRuntimeMethod Include="FS_readFile" />
```

Comment:
Is this not needed?

Reply:
emcc complains it's invalid when I set asserts on. 
It's used for symbolicator on JS side.
https://github.com/dotnet/runtime/blob/f604caf8d4014336bb04359cdfed70767f2975ae/src/mono/wasm/runtime/logging.ts#L190

Why is that in VFS @radical ?

Reply:
As long as this can read it, sure.

Reply:
Will it mean not early enough?

Reply:
VFS is like any other asset. If we need it before emscripten start (before wasm) we need some other approach.

Reply:
No, not earlier than what it is now.

Reply:
Right now we are not blocking emscripten start till after all VFS arrived.
We are blocking mono till after all assets arrived.

Reply:
done